src/utils/validation/postFormSchema.js

```diff
@@ -0,0 +1,12 @@
+const Joi = require('joi');
+
+const postFormSchema = Joi.object({
+  title: Joi.string().required(),
+  image_url: Joi.string().allow('')
+    .optional(),
+  content_post: Joi.string()
+    .allow('')
+    .optional(),
+});
```

Comment:
Customizing the error messages can enhance the user experience by providing clear and meaningful feedback when validation fails.

To add custom error messages to the `postFormSchema` validation schema, you can use the `messages` method provided by Joi. Here's an example of how you can define custom error messages:

```js
const Joi = require('joi');

const postFormSchema = Joi.object({
  title: Joi.string().required().messages({
    'any.required': 'Title is required.',
  }),
  image_url: Joi.string().allow('').optional(),
  content_post: Joi.string().allow('').optional(),
})

module.exports = postFormSchema;
```